.codacy_coverage.sh

```diff
@@ -1,6 +1,13 @@
 #First install the reporter tool
 wget -q -O ~/codacy-coverage-reporter-assembly-latest.jar https://oss.sonatype.org/service/local/repositories/releases/content/com/codacy/codacy-coverage-reporter/4.0.1/codacy-coverage-reporter-4.0.1-assembly.jar > /dev/null
 
+# Create all the .gcov files
+# Run gcov in the same directory as the source file for the main
+# library (because we used recursive make)
+find . -type f -name *.gcno -execdir gcov -pb -r {} +
```

Comment:
Should this be `find ./src` (or some list explicitly excluding the unit test directory)?

Reply:
It doesn't seem to make a difference in terms of what `.gcov` files get generated. I'll have to wait till codacy actually generates something for this PR to see if it affects that

Reply:
Ah, I'm not overly concerned about that, more worried about under-counting!